src/patternfly/components/Switch/switch.hbs

```diff
@@ -0,0 +1,10 @@
+<label for="{{id}}" class="pf-c-switch">
```

Comment:
can you refactor this hbs code so that each element is it's own partial, and each partial takes arguments of `partial-name--modifier` and `partial-name--attribute`? The [popover component](https://github.com/patternfly/patternfly-next/tree/master/src/patternfly/components/Popover) is a good example, but you can look at any component or layout to see the structure.